src/Controller/PrivacyDataAjax.php

```
@@ -140,20 +148,52 @@ public function createCSV($data, $filename)
             }
-            fputcsv($fp, array_values($fields));
+            fputcsv($fp, array_map(array(CsvHelper::class, 'neutralizeFormula'), array_values($fields)));
             $i++;
         }
         fclose($fp); // phpcs:ignore WordPress.WP.AlternativeFunctions.file_system_operations_fclose
         $file_url = $upload_dir['baseurl'] . '/' . $filename . '.csv';
 
+        // Schedule removal of the export shortly after it is generated so the
+        // personal data does not remain available in the uploads directory.
+        if (!wp_next_scheduled(self::CLEANUP_HOOK, array($filepath))) {
+            wp_schedule_single_event(time() + 15 * MINUTE_IN_SECONDS, self::CLEANUP_HOOK, array($filepath));
```

PHPStan (2.2.6)

[error] Constant MINUTE_IN_SECONDS not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)